docs/tutorials/01_introductory/viz_texture.py

```diff
@@ -0,0 +1,37 @@
+"""
+===============
+Sphere Texture
+===============
+In this tutorial, we will show how to create a sphere with a texture.
+"""
+
+import numpy as np
+from fury import window, actor, utils, primitive, io
+import itertools
+
+##############################################################################
+# Create a a scene to start.
+
+scene = window.Scene()
+
+##############################################################################
+# Load an image (png, bmp, jpeg or jpg) using ``io.prim_sphere``.
+
+filename = r"C:\Users\melin\Downloads\1_earth_8k.jpg"
```

Comment:
@mlraglin thanks for doing this. Improvements and examples are very welcome ! You may want to make the picture available as a FURY resource within this PR, and change the above line so that it is automatically fetched from there. Otherwise, other folks will not be able to replicate your example.

@skoudoro Not sure how examples get tested, but a minimal approach would be to at least run them., right? That may help in identifying issues.

Reply:
Hi @jhlegarreta,

We are running all the examples as you can see on [this build](https://travis-ci.com/github/fury-gl/fury/builds/166751149). And yes, the doc generation was failing. However, a long time ago, I added an option to allow this matrix to failed. Sometimes failures are not so obvious and I fix them later... 😄. 

Here some more details @mlraglin from @jhlegarreta good requests:
1. Looks at a fetcher [here](https://github.com/fury-gl/fury/blob/master/fury/data/fetcher.py#L291)
2. Create a new one named `fetch_viz_textures`
3. Add the fetcher to the [__init__ file here](https://github.com/fury-gl/fury/blob/master/fury/data/__init__.py)
4. add the new fetcher to your tutorial. you can look at an example [here](https://github.com/fury-gl/fury/blob/master/docs/examples/viz_network.py#L22-L24)
5. if you have any questions. Do not hesitate (here or [discord channel](https://discord.gg/6btFPPj))

Thanks @jhlegarreta for the feedback
Thanks @mlraglin for your last update

Reply:
@skoudoro thanks :+1: ! Sorry for not having had a closer look at the builds.

Reply:
@jhlegarreta Thank you for pointing this out! @skoudoro I have implemented the new fetcher ``fetch_viz_textures`` in the tutorial to solve this issue.

Reply:
@mlraglin good job :thumbsup:.